environmental_data_modules/met_module.py

```diff
@@ -17,7 +19,9 @@ def __init__(self):
                 Initialised instance of subclass of MetModule
 
         """
-        self._columns_base = MetModule.COLUMNS_BASE
+        self._timestamp_string = MetModule.TIMESTAMP_STRING
+        self._site_string = MetModule.SITE_STRING
+        self._columns_base = [MetModule.TIMESTAMP_STRING,MetModule.SITE_STRING]
         self._columns_specific = []
 
```

Comment:
Yes so having these as class properties is OK. (It could make sense to have these as static, but let's leave as is for now, as I'm pretty sure that follows a pattern that I've often used.)